src/assets/styles.css

```diff
@@ -0,0 +1,175 @@
+body 
+
+{
```

Comment:
CSS should be scoped to each component and placed in the `styled.ts` file in each component folder. Please use [the generator](https://redback-operations.github.io/redback-documentation/docs/web-mobile-app-dev/frontend/new-components) to create the proper files to start with.